src/test/java/guitests/PersonEditGuiTest.java

```diff
@@ -139,22 +136,45 @@ public void cancelOperation_usingAccelerator() {
         Person pandaWong = new PersonBuilder("Panda", "Wong")
                 .withStreet("Chengdu Panda Street").withCity("Chengdu").withPostalCode("PANDA")
                 .withBirthday("01.01.1979").withGithubUsername("panda").withTags(td.colleagues, td.friends).build();
-        addPersonDialog.enterNewValues(pandaWong);
-        addPersonDialog.clickOk();
-
-        personListPanel.clickOnListView(); // To ensure shortcut keys work properly.
+        addPersonDialog.enterNewValues(pandaWong).clickOk();
 
         personListPanel.use_LIST_GOTO_BOTTOM_SEQUENCE();
```

Comment:
Why is there is a need to go to the bottom here? If you simply want to go to a specific card, we should not use go_to_bottom as it is misleading to the reader.